packages/core/stories/about/Usage.stories.mdx

```diff
@@ -1,20 +1,40 @@
 import { Meta } from '@storybook/addon-docs/blocks';
 import { linkTo } from '@storybook/addon-links';
-import {
-  Button
-} from '@massds/mayflower-react';
+import { Button } from '@massds/mayflower-react';
 
 import generateTitle from '../util/generateTitle';
 const { STORYBOOK_CDN_PATH, STORYBOOK_CDN, STORYBOOK_PKG } = process.env;
 
 
 <Meta title={generateTitle('Usage')} />
 
-# Install Mayflower
+<h1>Get Started with Mayflower <small>version {process.env.VERSION}</small></h1>
+
+
 
 We encourage all Commonwealth web properties to implement the Mayflower Design System for a consistent look and feel. It’s important that constituents know they are on an official state government website, and the visual cues that come from using a cohesive design help convey that legitimacy.
 
-## Mayflower assets
+
+## Quick Start
+To kickstart an HTML page with Mayflower Header, Footer, basic layout and styles:
+<Button theme="c-primary-alt" onClick={linkTo(generateTitle('Template'))}>Start From a Template</Button>
+
+To start building with Mayflower in HTML, start by importing the global styles and then incrementally add the styles and JS for each Mayflower component:
+
+- To import fonts and get started with basic Mayflower styles, link to the compiled `css/global.css` (or the minified version, `css/global.min.css`) from the @massds/mayflower-assets package or the assets CDN.
+
+- To include basic page layout styles from Mayflower, link to the compiled `css/layout.css` (or the minified version, `css/layout.min.css`) from the @massds/mayflower-assets package or the assets CDN.
+
+- To use a Mayflower branding component on your site, go to the documentation of the component, copy and paste the HTML code snippet, and link to the CDN version of CSS and JS files in the appropriate sections of your HTML.
+
+> You should always link to the Mayflower CDN at a specific version. That way, you never have to worry about your website getting impacted by any Mayflower breaking updates, and you can upgrade to a new Mayflower version whenever your team is ready. Alternatively, you can host the CSS and JS downloaded from mayflower assets package. 
+
+> To view a specific version of this documentation, go to https://mayflower.digital.mass.gov/core/v/:version
```

Comment:
There is a storybook addon for versioning. Should we use that?
https://www.npmjs.com/package/storybook-addon-versions

Reply:
hmm this doesn't seem be well supported, it was published 3 years ago not sure if it works with the latest storybook